buildpipeline/pipeline.json

```diff
@@ -141,8 +140,7 @@
           "Parameters": {
             "PB_AdditionalBuildArguments": "/p:PortableBuild=true /p:SkipTests=true /p:NativeToolSetDir=C:\\tools\\clr",
             "PB_TargetArchitecture": "arm64",
-            "PB_PortableBuild": "true",
-            "PB_DistroRid": "win-arm64"
+            "PB_PortableBuild": "true"
```

Comment:
We would still need some way to explicitly call publish (for the Windows signed builds which are `build` broken into individual steps), but that should still be feasible with a property.  Also, if a build definition is not just calling build (like the Windows signing steps), then publish wouldn't automatically be included.

That being said, it may be too many moving pieces at the moment.  I could buy that argument.  There's a fine line to walk at the moment between expedience and improvements.